pandas/tests/plotting/frame/test_frame.py

```diff
@@ -148,6 +148,23 @@ def test_plot(self):
         result = ax.axes
         assert result is axes[0]
 
+    def test_nullable_int_plot(self):
+        # GH 32073
+        dates = ["2008", "2009", None, "2011", "2012"]
+        df = pd.DataFrame(
+            {
+                "A": [1, 2, 3, 4, 5],
+                "B": [7, 5, np.nan, 3, 2],
+                "C": pd.to_datetime(dates, format="%Y"),
+                "D": pd.to_datetime(dates, format="%Y", utc=True),
+            }
+        )
+
+        _check_plot_works(df.plot, x="A", y="B")
+        _check_plot_works(df[["A", "B"]].astype("Int64").plot, x="A", y="B")
```

Comment:
I think we'd rather to specify the dtype in the dataframe construction

Reply:
Sure. Thank you for the suggestion. I will rectify this in the next commit.

Reply:
@arw2019 I added the changes to the latest commit. Can you please review it again ?